qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -509,28 +546,105 @@ def _build_eom_matrices(
         logger.debug("\nM:=========================\n%s", m_mat)
         logger.debug("\nV:=========================\n%s", v_mat)
 
-        return m_mat, v_mat, q_mat, w_mat, m_mat_std, v_mat_std, q_mat_std, w_mat_std
+        # Matrix building rules
+        # h_mat = [[M, Q], [P, N]] and s_mat = [[V, W], [T, U]]
+        # N = M.conj() = M.T
```

Comment:
Is this the case by construction?

Reply:
Yes this comes from analytics. Note that in many cases the previous definitions worked as W is often the 0 matrix, and V is often real, but my comments describe the only expected properties of these matrices for arbitrary expansion operators (ie the most general case).